Documentation/network/kubernetes/kubeproxy-free.rst

```diff
@@ -1543,6 +1543,29 @@ For more information, ensure that you have the fix `Pull Request <https://github
     48498    getpeername4
     48494    getpeername6
 
+Known Issues
+############
+
+In case of clusters deployed with cilium version < 1.11.15, there was an issue
+discovered where service backend entries could be leaked in the BPF maps in some. The
+known corner cases that could lead to such leaks are due to race conditions between
+deletion of a service backend while it's terminating, and simultaneous deletion of the service
+the backend is associated with. This could lead to duplicate backend entries
+getting created that could eventually lead to fill up of the ``cilium_lb4_backends_v2`` map.
+You may see error messages like these in the cilium agent logs in such cases::
```

Comment:
Edits for plain language; capitalize "Cilium"
```suggestion
For clusters deployed with Cilium version < 1.11.15, service backend entries could be leaked in the BPF maps in some instances. The
known corner cases that could lead to such leaks are due to race conditions between
deletion of a service backend while it's terminating, and simultaneous deletion of the service
the backend is associated with. This could lead to duplicate backend entries
that could eventually fill up the ``cilium_lb4_backends_v2`` map.
In such cases, you might see error messages like these in the Cilium agent logs::
```